cli/src/main/resources/hudson/cli/client/Messages.properties

```diff
@@ -15,6 +15,7 @@ CLI.Usage=Jenkins CLI\n\
   \                                 for use with -http.\n\
   \                                 Passing credentials by file is recommended.\n\
   \                                 See https://jenkins.io/redirect/cli-http-connection-mode for more info and options.\n\
+  \ -bearer [ TOKEN | @FILE ]     : specify authentication using a bearer token (or load the token from file)\n\
```

Comment:
it is exclusive to `-auth`. Can we check it and say it explicitly on the help. Also `-bearer` is for -http I guess, and passing it with a file also applies.

Sorry for being picky, I was about to approve when you said it cannot have a :smile: but with the new way, I believe is worth to document it well.

Reply:
@MRamonLeon Thank you for the review :+1:  I have updated the message and added a warning when both are used (this is how -user and -auth exclusivity was handled already)